Library/Formula/netperf.rb

```diff
@@ -11,6 +11,12 @@ class Netperf < Formula
     sha256 "4e0aa192670d1de341e77e0bd3af4b2cccc6a5b12a734e922e92c7b7d50f281d" => :mountain_lion
   end
 
+  # patch fixes demo mode which is required by a lot of software that depends on netperf
+  patch :p2 do
+    url "https://raw.githubusercontent.com/trelane/netperf-brew-patch/master/netlib.diff"
```

Comment:
I'm not a developer and I rarely use git, that'd require at least an hour and a half of work.  It took me 5 hours to make this patch apply correctly.